Source/GRChomboCore/BoundaryConditions.hpp

```diff
@@ -97,11 +98,14 @@ class BoundaryConditions
     ProblemDomain m_domain; // the problem domain (excludes boundary cells)
     Box m_domain_box;       // The box representing the domain
     bool is_defined; // whether the BoundaryConditions class members are defined
+    int m_symmetry_factor;
 
   public:
     /// Default constructor - need to call define afterwards
     BoundaryConditions() { is_defined = false; }
 
+    int get_symm() const { return m_symmetry_factor; }
```

Comment:
Abbreviated names are against our conventions :-) Amend to `get_symmetry_factor()`